core/client/src/light/backend.rs

```diff
@@ -128,6 +128,14 @@ impl<S, F, Block, H> ClientBackend<Block, H> for Backend<S, F> where
 	fn revert(&self, _n: NumberFor<Block>) -> ClientResult<NumberFor<Block>> {
 		unimplemented!()
 	}
+
+	fn insert_aux<'a, 'b: 'a, 'c: 'a, I: IntoIterator<Item=&'a (&'c [u8], &'c [u8])>, D: IntoIterator<Item=&'a &'b [u8]>>(&self, _insert: I, _delete: D) -> ClientResult<()> {
+		unimplemented!()
```

Comment:
Probably `Err(ClientErrorKind::NotAvailableOnLightClient.into())`? Do we need this on light client at all?

Reply:
Not now at least. This will be used only by the consensus engine for now.